src/EditorFeatures/Core/Implementation/CodeActions/CodeActionEditHandlerService.cs

```diff
@@ -214,11 +214,13 @@ internal class CodeActionEditHandlerService : ForegroundThreadAffinitizedObject,
             if (projectChange.GetAddedAdditionalDocuments().Any() ||
                 projectChange.GetAddedAnalyzerReferences().Any() ||
                 projectChange.GetAddedDocuments().Any() ||
+                projectChange.GetAddedAnalyzerConfigDocuments().Any() ||
                 projectChange.GetAddedMetadataReferences().Any() ||
                 projectChange.GetAddedProjectReferences().Any() ||
                 projectChange.GetRemovedAdditionalDocuments().Any() ||
                 projectChange.GetRemovedAnalyzerReferences().Any() ||
                 projectChange.GetRemovedDocuments().Any() ||
+                projectChange.GetRemovedAnalyzerConfigDocuments().Any() ||
```

Comment:
any chance we could extract this out into separate PRs to make this easier to review?

Reply:
I really wanted to extract these changes into a separate PR, but I wasn't able to verify the end-to-end functionality of code fix application, preview, opening these documents in editor, etc. without using the configure severity code action added in this PR. I can divide these into 2 PRs if this still seems critical for easier review.

Reply:
It's 7k lines :)  Anythin to make it more manageable would be appreciated :D   Up to you though.  I can def get through this.  it'll just take a while.